makedeb.sh

```diff
@@ -211,7 +218,9 @@ function build_opennms()
     echo
     echo "Version: " $VERSION
     echo "Release: " $RELEASE
+    echo "OPA VERSION: " $OPA_VERSION
     echo
+    sed -i "s/OPA_VERSION/$OPA_VERSION/g" debian/control
```

Comment:
Not real excited about changing "source" files during the build. I wonder if this could be a `control.in` file instead and we translate it in `debian/rules` or something.